### PR TITLE
Update to Spectrum Tokens Beta 17

### LINKED ISSUE #230: Dismissible Dialogs have incorrectly positioned close button on Large scale
### Steps to reproduce

1. Go to http://opensource.adobe.com/spectrum-css/2.13.0/docs/?color=light&scale=large&method=standalone#dialog-dialog-dismissible
2. Select Large scale in the dropdown at the top
3. Observe

### Spectrum-CSS version

2.13.0

#### Screenshots (if applicable)

image

image

### LINKED ISSUE #517: Accordion item heading height, icon spacing, and font style don't match designs
## Description

Accordion's heading doesn't match the designs.

## Steps to reproduce

1. Go to http://spectrum-css-archives.s3-website.us-east-2.amazonaws.com/2.16.0/docs/accordion.html
2. Screenshot it
3. Compare to XD file here https://www.dropbox.com/sh/6t8so9becymkf55/AACsFVyajcgnOPhAst-bm5hfa?dl=0
4. Note the various mismatches: height, font style, icon spacing

## Expected behavior

Font style and spacing between Chevron match

## Screenshots

image

## Environment
 - **Spectrum CSS version:** 2.16.0
 - **Browser(s) and OS(s):** Chrome on macOS

## Additional context

Discovered while testing #511

### PR DESCRIPTION
**Pull request by GarthDB**
## Description
I updated the latest Spectrum Tokens package (beta 17) and found it broke a lot of components.
Most likely it's because a bunch of deprecated tokens was deleted. I need to figure out how they were deleted. 

## Closes issues

Fixes #230
Fixes #517 

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

### CODE REVIEW

File: components/vars/css/components/spectrum-badge.css [hunk 1]
```diff
@@ -1,6 +1,5122 @@
 :root {
-  --spectrum-badge-text-color: var(--spectrum-global-color-static-white);
-  --spectrum-badge-error-background-color: var(--spectrum-semantic-negative-color-default);
-  --spectrum-badge-info-background-color: var(--spectrum-semantic-informative-color-default);
-  --spectrum-badge-warning-background-color: var(--spectrum-semantic-notice-color-default);
+  --spectrum-badge-fuchsia-l-focus-ring-border-radius-key-focus: var(--spectrum-global-dimension-size-75);
```
Comment (jluyau) [inline components/vars/css/components/spectrum-badge.css:2]: Is it required that the css variables are now a combination of color and size? Was looking at the previous label css and it looked as if there were separate variables for sizing and colors.

Comment (lazd) [reply components/vars/css/components/spectrum-badge.css:2]: Hmm, this looks wrong. @GarthDB thoughts?

Comment (jluyau) [reply components/vars/css/components/spectrum-badge.css:2]: Any updates on this? @GarthDB @lazd

Comment (GarthDB) [reply components/vars/css/components/spectrum-badge.css:2]: @jluyau the CSS vars file has the size and color tokens. What's new in `spectrum-badge` file are the t-shirt sizes. T-shirt sizes are separate from `scale` (i.e., `large`, `medium`). We are going to see a ton of new tokens for each component that gets t-shirt sizing.

Comment (jluyau) [reply components/vars/css/components/spectrum-badge.css:2]: Thanks for the update @GarthDB

**Comment by GarthDB** [comment]
- [ ] dropindicator
- [ ] sidenav
- [ ] accordion
- [ ] dropzone
- [ ] site
- [ ] actionbar
- [ ] fieldgroup
- [ ] slider
- [ ] actiongroup
- [ ] fieldlabel
- [ ] splitbutton
- [ ] actionmenu
- [ ] icon
- [ ] splitview
- [ ] asset
- [ ] illustratedmessage
- [ ] statuslight
- [ ] assetlist
- [ ] inputgroup
- [ ] steplist
- [ ] avatar
- [ ] label
- [ ] stepper
- [ ] breadcrumb
- [ ] link
- [ ] switch
- [ ] button
- [ ] menu
- [ ] table
- [ ] buttongroup
- [ ] miller
- [ ] tabs
- [ ] calendar
- [ ] modal
- [ ] tags
- [ ] card
- [ ] overlay
- [ ] textfield
- [ ] checkbox
- [ ] page
- [ ] thumbnail
- [ ] coachmark
- [ ] pagination
- [ ] toast
- [ ] colorarea
- [ ] picker
- [ ] tooltip
- [ ] colorhandle
- [ ] popover
- [ ] tray
- [ ] colorloupe
- [ ] progressbar
- [ ] treeview
- [ ] colorslider
- [ ] progresscircle
- [ ] typography
- [ ] colorwheel
- [ ] quickaction
- [ ] underlay
- [ ] commons
- [ ] radio
- [ ] vars
- [ ] cyclebutton
- [ ] rating
- [ ] well
- [ ] dialog
- [ ] search
- [ ] divider
- [ ] searchwithin

**Pull request merged**